_studio/mfx_lib/decode/av1/src/mfx_av1_dec_decode.cpp

```diff
@@ -924,45 +933,29 @@ mfxStatus VideoDECODEAV1::SubmitFrame(mfxBitstream* bs, mfxFrameSurface1* surfac
                     sts = MFX_ERR_GPU_HANG;
                     break;
 #endif
-            }
-
-            if (umcRes == UMC::UMC_WRN_REPOSITION_INPROGRESS)
-            {
-                if (!m_first_run)
-                {
-                    sts = MFX_WRN_VIDEO_PARAM_CHANGED;
-                }
-                else
-                {
-                    umcRes = UMC::UMC_OK;
-                    m_first_run = false;
-                }
-            }
 
-            if (umcRes == UMC::UMC_OK && m_allocator->FindFreeSurface() == -1)
-            {
-                sts = MFX_ERR_MORE_SURFACE;
-                umcFrameRes = UMC::UMC_ERR_NOT_ENOUGH_BUFFER;
-            }
+                case UMC::UMC_ERR_NOT_ENOUGH_BUFFER:
+                case UMC::UMC_WRN_INFO_NOT_READY:
+                case UMC::UMC_ERR_NEED_FORCE_OUTPUT:
+                    force = (umcRes == UMC::UMC_ERR_NEED_FORCE_OUTPUT);
```

Comment:
This force variable is not actually used anywhere.... delete...

Reply:
Done